helm_deploy/values-staging.yaml

```diff
@@ -0,0 +1,36 @@
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 4
+
+image:
+  repository: quay.io/hmpps/probation-offender-search
+  tag: latest
+  port: 8080
+
+ingress:
+  enabled: true
+  enable_whitelist: true
+  annotations:
+    kubernetes.io/ingress.class: "nginx"
+  hosts:
+    - host: probation-offender-search-staging.hmpps.service.justice.gov.uk
+      cert_secret: probation-offender-search-cert
+  path: /
+
+env:
+  SPRING_SECURITY_OAUTH2_RESOURCESERVER_JWT_JWK_SET_URI: "https://sign-in-preprod.hmpps.service.justice.gov.uk/auth/.well-known/jwks.json"
+  COMMUNITY_ENDPOINT_URL: "https://community-api-secure.stage.delius.probation.hmpps.dsd.io"
+  JAVA_OPTS: "-Xmx512m"
+
+whitelist:
+  office: "217.33.148.210/32"
+  health-kick: "35.177.252.195/32"
+  mojvpn: "81.134.202.29/32"
+  cloudplatform-live1-1: "35.178.209.113/32"
+  cloudplatform-live1-2: "3.8.51.207/32"
+  cloudplatform-live1-3: "35.177.252.54/32"
+  delius-aws-1: "3.10.104.193/32"
```

Comment:
I guess these are taken from the Confluence page I gave you

Reply:
yes